packages/manager/cypress/e2e/core/firewalls/update-firewall.spec.ts

```diff
@@ -396,4 +396,62 @@ describe('update firewall', () => {
         });
     });
   });
+
+  /*
+   * - Confirms that firewall's label can be updated on landing page'.
+   */
+  it("updates a firewall's label", () => {
+    const region = chooseRegion();
+
+    const linodeRequest = createLinodeRequestFactory.build({
+      label: randomLabel(),
+      region: region.id,
+      root_pass: randomString(16),
+    });
+
+    const firewallRequest = firewallFactory.build({
+      label: randomLabel(),
+      rules: {
+        inbound: [],
+        outbound: [],
+      },
+    });
+
+    const newFirewallLabel = randomLabel();
+
+    cy.defer(
+      createLinodeAndFirewall(linodeRequest, firewallRequest),
+      'creating Linode and firewall'
+    ).then(([_linode, firewall]) => {
+      cy.visitWithLogin('/firewalls');
+
+      // Confirm that firewall is listed on landing page with expected configuration.
+      cy.findByText(firewall.label)
+        .closest('tr')
+        .within(() => {
+          cy.findByText(firewall.label).should('be.visible');
+          cy.findByText('Enabled').should('be.visible');
+          cy.findByText('No rules').should('be.visible');
+          cy.findByText('None assigned').should('be.visible');
+        });
+
+      cy.visitWithLogin(`/firewalls/${firewall.id}`);
+
+      cy.get(`[aria-label="Edit ${firewall.label}"]`).click();
```

Comment:
```suggestion
      cy.findByLabelText(`Edit ${firewall.label}`).click();
```
Slightly more clear this way!